engines/python/setup/djl_python/deepspeed.py

```diff
@@ -236,17 +301,71 @@ def infer_task_from_model_architecture(self, config: PretrainedConfig):
                 f"Task could not be inferred from model config. "
                 f"Please manually set `task` in serving.properties.")
 
+    def get_model_pretrained(self, model_id_or_path, torch_dtype='auto', **kwargs):
+        tokenizer = AutoTokenizer.from_pretrained(model_id_or_path)
+        model = TASK_TO_MODEL[self.task].from_pretrained(model_id_or_path, torch_dtype=torch_dtype, **kwargs)
+        return model, tokenizer
+
+    def get_model_from_config(self, model_id_or_path, **kwargs):
+        tokenizer = AutoTokenizer.from_pretrained(model_id_or_path)
+        model = TASK_TO_MODEL[self.task].from_config(self.model_config, **kwargs)
+        return model, tokenizer
+
+    def get_model(self, model_id_or_path, loading_method, **kwargs):
+        if loading_method == 'from_config':
+            return self.get_model_from_config(model_id_or_path, **kwargs)
+        elif loading_method == 'pretrained':
```

Comment:
this logic seemed to be the same? just 

```
if loading_method in ['from_config', 'pretrained']:
```

Reply:
Oh man, good catch. line 316 should call `self.get_model_pretrained`
Will fix this typo in next iteration.